roles/keystone/tasks/main.yml

```diff
@@ -80,6 +87,17 @@
   service: name=keystone state=stopped
   when: database_create.changed or force_sync|default('false')|bool
   tags: db-migrate
+  
+- name: disable keystone apache site before db sync
+  apache2_site: name=keystone state=disabled
+  when: database_create.changed or force_sync|default('false')|bool
+  tags: db-migrate
```

Comment:
This kills off the apache front end, the task above this kills off the uwsgi procs.